sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -9,16 +9,19 @@ The currently supported targets are all Intel GPUs starting with Gen9.
 
 This extension provides a feature-test macro as described in the core SYCL specification section 6.3.3 "Feature test macros". Therefore, an implementation supporting this extension must predefine the macro SYCL_EXT_ONEAPI_BACKEND_LEVEL_ZERO to one of the values defined in the table below. Applications can test for the existence of this macro to determine if the implementation supports this feature, or applications can test the macro’s value to determine which of the extension’s APIs the implementation supports.
 
+NOTE: By necessity, this specification exposes some details about the way SYCL is layered on top of the Level Zero backend.  Although the DPC++/SYCL implementers make every effort to retain backward compatibility for changes to this specification, it is occasionally necessary to change an API in this specification in a way that is not compatible with previous versions. For example, this sometimes happens when DPC++ changes the way it layers SYCL on top of Level Zero.  When an API breaking change like this occurs, we update the version of the SYCL_EXT_ONEAPI_BACKEND_LEVEL_ZERO feature-test macro and describe the API breakage as part of the description for that API version.
 
 |Value|Description|
 |---|:---|
 |1|Initial extension version.
 |2|Added support for the make_buffer() API.
 |3|Added device member to backend_input_t<backend::ext_oneapi_level_zero, queue>.
+|4|Changed backend_input_t<backend::ext_oneapi_level_zero, queue> and backend_return_t<backend::ext_oneapi_level_zero, queue>.

```

Comment:
I'd suggest slightly different wording:

> Change the definition of `backend_input_t` and `backend_return_t` for the `queue` object, which changes the API for `make_queue` and `get_native` (when applied to `queue`).

This provides a better summary of the APIs that changed in this version.  I think we don't need to be so precise about the exact template parameters to `backend_input_t` and `backend_return_t` here.  Those details are described more precisely in the main part of this spec.